cpp/ScanChain_Zp.C

```diff
@@ -210,7 +213,7 @@ int ScanChain(TChain *ch, double genEventSumw, TString year, TString process) {
 	      bool mu_trk_and_global = ( nt.Muon_isGlobal().at(mu) && nt.Muon_isTracker().at(mu) );
 	      bool mu_id = ( nt.Muon_highPtId().at(mu) == 2 );
 	      bool mu_pt_pf = ( nt.Muon_pt().at(mu) > 53 && fabs(nt.Muon_eta().at(mu)) < 2.4 );
-	      bool mu_relIso = ( nt.Muon_tkRelIso().at(mu) < 0.1 );
+	      bool mu_relIso = ( nt.Muon_tkRelIso().at(mu) < 0.02 );
```

Comment:
Is this an intended change? Even though it is a change in our baseline selection, it is not announced in the PR description. Moreover, it is changed to the value of the `ΔR(trigger, muon)`, so that makes me wonder if it was changed by mistake.

Taking a look at the distributions, maybe tightening this isn't a bad choice but `0.02` seems a bit too tight for now.

Reply:
No, it wasn't intentional. Thanks for catching it!
Fixed.